.eslintrc.js

```diff
@@ -1,10 +1,3 @@
-const fs = require("fs");
```

Comment:
For context the way eslint worked in the project was....
 
This file was the base config and contained all of the common configs and rules for each directory. 

The one in the `/src` directory extended this config. And applied its directory-specific rules on top of it.

The one in the `/cypress` directory also extended this config and applied its directory-specific rules on top of it.